frontend/sw360-portlet/src/main/webapp/html/moderation/view.jsp

```diff
@@ -104,85 +122,140 @@
-                columns: [
-                    {"title": "Document Name"},
-                    {"title": "Requesting User"},
-                    {"title": "Moderators"},
-                    {"title": "State"},
-                    {"title": "Actions"}
-                ]
-            });
+    function createModerationsTable(tableId, tableData) {
+        var tbl = $(tableId).DataTable({
+            pagingType: "simple_numbers",
+            dom: "lrtip",
+            data: tableData,
+            autowidth: false,
+            columns: [
+                {title: "Date", render: {display: renderTimeToReadableFormat}},
+                {title: "Document Name"},
+                {title: "Requesting User"},
+                {title: "Group"},
+                {title: "Moderators", render: {display: renderModeratorsListExpandable}},
+                {title: "State"},
+                {title: "Actions"}
+            ]
+        });
 
-            return tbl;
-        }
+        return tbl;
+    }
 
-        function deleteModerationRequest(id, docName) {
-            function deleteModerationRequestInternal() {
-                jQuery.ajax({
-                    type: 'POST',
-                    url: '<%=deleteModerationRequestAjaxURL%>',
-                    cache: false,
-                    data: {
-                        <portlet:namespace/>moderationId: id
-                    },
-                    success: function (data) {
-                        if (data.result == 'SUCCESS') {
-                            closedModerationsDataTable.row('#' + id).remove().draw(false);
-                        }
-                        else {
-                            $.alert("I could not delete the moderation request!");
-                        }
-                    },
-                    error: function () {
+    function deleteModerationRequest(id, docName) {
+        function deleteModerationRequestInternal() {
+            jQuery.ajax({
+                type: 'POST',
+                url: '<%=deleteModerationRequestAjaxURL%>',
+                cache: false,
+                data: {
+                    <portlet:namespace/>moderationId: id
+                },
+                success: function (data) {
+                    if (data.result == 'SUCCESS') {
+                        closedModerationsDataTable.row('#' + id).remove().draw(false);
+                    }
+                    else {
                         $.alert("I could not delete the moderation request!");
                     }
-                });
-            }
+                },
+                error: function () {
+                    $.alert("I could not delete the moderation request!");
+                }
+            });
+        }
 
             confirm.confirmDeletion("Do you really want to delete the moderation request for <b>" + docName + "</b> ?", deleteModerationRequestInternal);
         }
     });
+
+    function getEmailFromLink(link) {
```

Comment:
I do not understand how the name of this function relates to its code. Kindly explain

Reply:
@alexbrdn  This function extracts the email address from a HTML element generated by `org.eclipse.sw360.portal.users.UserUtils.displayUser`. I use this function because the displayed email address should be limited to a certain number of characters and I think that it is nicer to have clickable mailto-links only if the full list of moderators is shown in the moderation listing. But I agree with you that the name was not good. Changed it to `extractEmailFromHTMLElement`.